brands/shop/curtain.json

```
@@ -0,0 +1,16 @@
+{
+  "shop/curtain|Heytens": {
+    "countryCodes": [
+      "be",
+      "fr",
+      "lu",
+      "ch"
+    ],
+    "tags": {
+      "brand": "Heytens",
+      "brand:wikidata": "Q90565231",
+      "name": "Heytens",
+      "shop": "curtains"
```

This should be "curtain" to match the filename and the key.  
https://taginfo.openstreetmap.org/tags/shop=curtain
```suggestion
      "shop": "curtain"
```